src/MongoDB.Driver.Core/Core/Configuration/ConnectionPoolSettings.cs

```diff
@@ -93,6 +93,7 @@ public int MinConnections
         /// <value>
         /// The size of the wait queue.
         /// </value>
+        [Obsolete("This functionality will be replaced in next releases.")]
```

Comment:
Let's say "property" instead of "functionality" and "a later release" instead of "next releases".

    [Obsolete("This property will be removed in a later release.")]

"property" is more accurate and it will only be removed once so "a later release" is singular instead of plural.

Find all other instances and make the same change.